bin/_mocha

```diff
@@ -187,13 +187,44 @@ Error.stackTraceLimit = Infinity; // TODO: config
 
 var reporterOptions = {};
 if (program.reporterOptions !== undefined) {
-    program.reporterOptions.split(",").forEach(function(opt) {
-        var L = opt.split("=");
-        if (L.length != 2) {
-            throw new Error("invalid reporter option '" + opt + "'");
+
+   program.reporterOptions.split(",").forEach(
+
+    // Two possible formats ...
+     /\w+{(\w+:\w+)+}/.test(program.reporterOptions) ?
```

Comment:
> The original syntax still works, but if multiple reporters are present, it will apply that configuration to "all" active reporters.

Then it's not breaking. (?)